services/QuillLMS/app/models/district.rb

```diff
@@ -63,7 +63,57 @@ def vitally_data
         phone: phone,
         total_students: total_students,
         total_schools: total_schools
-      }
+      }.merge(vitally_diagnostic_rollups)
```

Comment:
Since the elements of `vitally_diagnostic_rollups` are conceptual siblings to the other members of this hash, it makes sense to use the double asterisk expansion syntax within this hash literal. Open to other opinions if you have them, though.